Adding ability to change environment type (#71)
AdobeDocs/commerce-events: src/pages/events/configure-commerce.md

1. Copy the ID returned in the command output into the **Adobe I/O Event Provider ID** field in the Admin.

> 1. Set the **Adobe I/O Environment** to be **Production** (if different)

Reviewer: ```suggestion
1. Set **Adobe I/O Environment** to `Production`.
```

> 1. Enable Commerce Eventing by changing the value of field **Enabled** to **Yes**

Reviewer: ```suggestion
1. Enable Commerce Eventing by setting **Enabled** to `Yes`.
```

Reviewer: I can't directly comment on line 166, since it wasn't changed in this PR, but please change the last word, Production, from bold to backticks.
